lib/graphql/all_organizations.graphql

```diff
@@ -0,0 +1,16 @@
+query AllOrgs{
```

Comment:
this and the create_app aren't being used yet here right?

Reply:
Yes they are related to my other PR for the create command but I was using them in this PR for tophatting purposes